exec/java-exec/src/main/java/org/apache/drill/exec/compile/ClassBuilder.java

```diff
@@ -141,7 +142,9 @@ public ClassBuilder(DrillConfig config, OptionManager optionManager) {
     // A key advantage of this method is that the code can be
     // saved and debugged, if needed.
 
-    saveCode(code, name);
+    if (cg.persistCode()) {
```

Comment:
I found "persistCode()" a bit confusing. From the method name, I thought it's the method to persist the code to directory, but looks like it's the method to check if we want to persistCode; the actual one is saveCode().